src/Features/CSharp/Portable/Completion/KeywordRecommenders/InKeywordRecommender.cs

```diff
@@ -36,7 +36,7 @@ private static bool IsInParameterModifierContext(int position, CSharpSyntaxConte
             if (context.SyntaxTree.IsParameterModifierContext(
                     position, context.LeftToken, includeOperators: true, out var parameterIndex, out var previousModifier))
             {
-                if (previousModifier == SyntaxKind.None)
+                if (previousModifier is SyntaxKind.None or SyntaxKind.ScopedKeyword)
```

Comment:
this feels odd.  `IsParameterModifierContext` should understand `Scoped` itself right?

Reply:
@CyrusNajmabadi I'm not sure what you mean?

Reply:
oh, misread.  sorry.

Reply:
can you say `this scoped`?

Reply:
I think yes. `this scoped` is valid. I'll add tests for `this scoped`, `this ref`, `this in`, `this scoped ref`, and `this scoped in`.